pkg/network/address.go

```diff
@@ -56,82 +47,165 @@ func GetMultiAddressesForHostQuiet(host host.Host) []multiaddr.Multiaddr {
 	return ma
 }
 
-// GetPriorityAddress returns the best public or private IP address
-func GetPriorityAddress(addrs []multiaddr.Multiaddr) multiaddr.Multiaddr {
-	bestAddr := getBestPublicAddress(addrs)
-	if bestAddr != nil {
-		return bestAddr
+// getPublicMultiAddress returns the best public IP address
+func getPublicMultiAddress(addrs []multiaddr.Multiaddr) multiaddr.Multiaddr {
```

Comment:
I'm just wondering here, why we still need to have to get the public IP? aren't we dialing now with libp2p?